concepts/admin.py

```diff
@@ -8,16 +11,46 @@ def resolve(modeladmin, request, queryset):
         authorities.resolve(type(obj), obj)
 resolve.verbose_name = 'resolve selected concepts'
 
+def merge_concepts(modeladmin, request, queryset):
+
+    opts = modeladmin.model._meta
+    app_label = opts.app_label
+    RESOLVED_STATUS = 'Resolved'
+    resolvedCount = 0
+    resolvedConcept = None
+
+    for item in queryset:
+        if item.concept_state == RESOLVED_STATUS:
+            resolvedCount += 1
+            resolvedConcept = item
+
+    context = {
+        "resolvedConcept":resolvedConcept,
+        "opts": opts,
+        "app_label": app_label
+    }
+
+    #VGNWB-121 gets called only when there is only one resolved concept
+    if resolvedCount == 1:
+        return render_to_response('admin/merge_concepts_resolved.html', context)
+
+    return render_to_response('admin/merge_concepts.html', {'concepts': queryset})
+merge_concepts.verbose_name = "Merge Concepts"
```

Comment:
``verbose_name`` shouldn't have caps in it; also, not really necessary since the method is already called ``merge_concepts`` (which will display as "merge concepts").